src/api-response.ts

```diff
@@ -11,8 +11,11 @@ export interface ApiResponse<S extends z.ZodTypeAny> {
   schema: S;
   /** @default 200 for a positive and 400 for a negative response */
   statusCode?: number | [number, ...number[]];
-  /** @default "application/json" */
-  mimeType?: string | [string, ...string[]];
+  /**
+   * @example [] for no content, such as 204 and 302
+   * @default "application/json"
+   * */
+  mimeType?: string | string[];
```

Comment:
what would it take to make it `null` instead of empty array?
I think it could be more intuitively understandable

Reply:
addressed in #2193

Reply:
fixed in 77731539